src/Components/Header.js

```diff
@@ -6,25 +6,27 @@ const Header = () => {
     return (
         <>
             <h2> Cat Tinder!</h2>
+           <div className='header-nav'>
             <header>
                 <Nav>
                     <NavItem>
                         <NavLink to="/">
-                            Home
+                            <h6> Home </h6>
```

Comment:
Double check that your test for Header still passes with this.  Nesting another element in your NavLink may cause your test to fail.  I would recommend styling the NavLink itself by adding a className and setting the style in CSS.